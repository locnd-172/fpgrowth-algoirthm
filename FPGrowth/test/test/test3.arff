@relation retail

@attribute 1076 {true}
@attribute 1133 {true}
@attribute 1164 {true}
@attribute 1361 {true}
@attribute 1417 {true}
@attribute 1674 {true}
@attribute 1689 {true}
@attribute 2182 {true}
@attribute 2298 {true}
@attribute 2791 {true}
@attribute 2873 {true}
@attribute 3121 {true}
@attribute 3182 {true}
@attribute 3221 {true}
@attribute 3383 {true}
@attribute 3559 {true}
@attribute 3596 {true}
@attribute 3639 {true}
@attribute 3700 {true}
@attribute 4191 {true}
@attribute 4324 {true}
@attribute 4726 {true}
@attribute 4747 {true}
@attribute 4789 {true}
@attribute 5030 {true}
@attribute 5276 {true}
@attribute 5624 {true}
@attribute 5882 {true}
@attribute 5951 {true}
@attribute 6242 {true}
@attribute 6357 {true}
@attribute 6440 {true}
@attribute 6601 {true}
@attribute 6862 {true}
@attribute 7010 {true}
@attribute 7113 {true}
@attribute 7272 {true}
@attribute 7310 {true}
@attribute 7662 {true}
@attribute 7740 {true}
@attribute 7804 {true}
@attribute 7993 {true}
@attribute 8085 {true}
@attribute 8283 {true}
@attribute 8312 {true}
@attribute 8651 {true}
@attribute 8810 {true}
@attribute 8953 {true}
@attribute 9771 {true}
@attribute 9856 {true}

@data

true,true,true,?,?,?,true,?,true,true,?,true,?,true,?,true,true,?,?,?,?,true,?,?,true,?,?,?,?,?,true,true,true,true,true,?,?,true,?,?,?,true,true,?,?,?,?,?,true,?
true,?,?,?,?,true,?,true,?,true,true,true,?,?,?,true,?,?,true,?,true,?,true,true,?,?,?,?,?,?,?,?,?,?,true,true,true,?,true,?,?,?,?,true,true,?,true,true,true,true
?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?
?,?,true,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,true,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
true,?,?,true,?,?,true,?,true,?,?,?,?,true,true,?,?,?,true,true,true,true,?,true,?,?,true,?,true,?,?,true,true,?,?,true,true,true,true,?,?,true,?,true,?,?,?,?,true,true
?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?
true,true,true,true,?,?,?,?,?,?,?,true,?,?,?,true,?,?,?,true,?,?,?,true,true,?,?,true,?,?,?,?,true,true,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?
?,?,true,?,true,true,?,?,?,?,?,?,?,true,?,?,true,true,?,?,?,?,?,?,true,?,true,?,?,?,?,?,?,true,true,?,true,true,true,?,?,?,true,true,true,?,true,true,?,true
?,?,?,true,?,?,?,?,?,?,?,?,true,?,?,true,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
true,true,?,true,?,?,?,true,?,?,?,true,?,true,?,?,?,true,true,?,?,true,?,true,?,true,?,?,?,?,?,?,true,true,?,?,true,?,?,?,true,?,?,?,true,true,?,?,?,?
?,?,?,?,?,?,?,?,?,true,?,?,?,?,true,?,true,?,?,?,?,?,?,?,true,?,?,?,?,?,true,?,?,?,?,?,?,?,true,?,?,?,?,true,?,?,?,?,true,?
?,?,true,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,true,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,true,?,true
?,?,?,true,?,true,true,true,?,?,true,?,?,true,?,true,?,true,true,?,?,true,true,true,true,true,?,true,?,?,true,true,true,?,?,?,true,?,true,?,?,true,?,true,?,?,?,?,?,true
true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
?,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
?,?,?,?,?,true,true,?,true,?,true,true,?,true,true,?,?,true,?,?,true,?,?,?,?,true,?,true,?,true,?,true,true,true,?,?,?,true,?,?,true,?,?,?,?,true,true,?,?,true
?,?,true,?,true,true,?,?,?,?,?,?,true,true,true,true,?,?,true,?,true,?,true,?,?,?,true,true,?,true,?,?,true,true,true,?,?,?,true,?,?,true,true,true,?,?,?,true,?,true
?,?,?,?,?,?,?,?,true,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,true,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
true,?,?,?,?,?,?,?,?,true,?,?,true,true,?,true,?,?,?,true,?,?,?,true,?,true,?,?,?,true,?,true,true,?,?,true,?,?,?,?,true,?,?,?,?,?,true,?,true,?
true,?,?,?,true,?,true,true,true,?,?,true,?,?,true,?,?,?,true,?,true,true,?,true,?,true,true,?,?,?,?,true,?,?,?,?,true,?,?,?,?,?,?,true,true,?,?,?,true,?
true,true,?,true,?,true,true,?,?,?,?,?,true,true,true,?,?,?,?,true,?,true,?,true,?,?,?,?,?,true,?,?,?,?,true,?,?,true,?,?,?,?,?,?,?,?,true,?,?,true
?,?,true,?,true,?,true,?,?,?,true,?,?,?,true,?,?,true,true,?,?,?,true,true,?,?,?,true,true,?,true,?,true,true,?,?,?,true,true,?,true,true,true,true,true,?,?,?,?,?
?,true,true,?,?,?,true,?,?,true,true,true,?,true,true,true,true,?,?,?,true,?,true,?,?,?,?,?,?,?,?,?,?,?,true,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?
true,?,?,?,?,?,true,?,?,?,?,?,?,?,?,true,true,true,true,?,true,?,?,?,?,true,?,true,?,true,true,true,?,true,?,true,?,true,true,true,?,true,?,true,?,?,?,?,true,true
?,?,true,?,true,?,?,?,true,true,?,?,true,?,?,true,?,true,?,?,true,?,?,true,true,true,?,?,true,?,?,?,true,?,?,true,true,true,true,?,?,?,?,?,true,?,?,?,?,?
?,true,?,true,true,?,?,?,?,?,?,true,?,?,?,true,?,?,?,?,?,?,true,?,true,true,true,?,?,true,?,true,?,true,?,?,?,?,true,?,?,?,?,?,?,?,true,?,true,?
?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,true,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?
?,true,?,?,?,true,?,?,?,?,true,?,true,?,true,?,?,?,?,?,?,?,?,?,?,true,true,?,true,true,?,?,true,?,?,?,?,?,true,?,?,true,?,?,true,?,?,?,?,true
