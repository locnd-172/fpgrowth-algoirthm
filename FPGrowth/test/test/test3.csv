1,5951
1,4324
1,5030
2,1076
2,6601
2,3221
2,6242
2,4191
2,9771
2,7804
2,7113
2,4789
2,3182
2,8810
2,2791
2,3559
2,6440
2,5276
3,8312
3,2298
3,4726
3,7272
3,5276
3,2182
3,3700
3,3383
3,6440
3,9771
3,3121
3,4789
3,1076
3,5624
3,1417
3,1689
3,4324
3,8283
4,4191
4,3182
4,7310
4,1076
4,9856
4,4726
4,4789
4,7010
4,6242
4,1133
4,1689
4,3383
4,8810
4,1674
4,1361
4,3221
5,5951
5,3383
5,1689
5,7662
5,5882
5,6601
5,1164
5,3639
5,1417
5,7804
5,7993
5,3700
5,6357
5,6862
5,8283
5,8085
5,8312
5,7310
5,2873
5,4747
5,4789
6,3121
6,4747
6,2791
6,2873
6,3383
6,3596
6,3221
6,1133
6,4324
6,1689
6,7272
6,7010
6,1164
6,3559
7,3596
7,7662
7,5276
7,6440
7,1689
7,8283
7,7993
7,6862
7,7113
7,9771
7,7310
7,6242
7,3700
7,1076
7,6357
7,9856
7,4324
7,5882
7,3639
7,3559
7,7740
8,2298
8,3639
8,8312
8,6601
8,5951
8,3559
8,5276
8,2791
8,7272
8,7662
8,1417
8,4324
8,4789
8,7310
8,5030
8,3182
8,7113
8,1164
9,7662
9,1133
9,9771
9,5624
9,8810
9,5030
9,3121
9,3559
9,1417
9,4747
9,6242
9,6440
9,5276
9,1361
9,6862
10,3700
10,8651
10,8312
10,5276
10,1361
10,2182
10,3639
10,7272
10,6862
10,4726
10,3221
10,1133
10,3121
10,7804
10,1076
10,4789
10,6601
11,3596
11,8283
11,5030
11,3383
11,2791
11,9771
11,6357
11,7662
12,4726
12,8312
12,9856
12,8953
12,1164
12,6242
12,1361
12,5030
13,5882
13,5276
13,4726
13,1361
13,7993
13,3221
13,7662
13,6357
13,5030
13,1689
13,3639
13,1674
13,6601
13,3700
13,8283
13,4789
13,3559
13,2873
13,6440
13,7272
13,4747
13,2182
13,9856
14,1076
14,7010
15,2873
16,2873
16,6601
16,3121
16,3383
16,7310
16,7804
16,6242
16,6862
16,6440
16,5882
16,3221
16,8651
16,8810
16,9856
16,2298
16,3639
16,4324
16,5276
16,1674
16,1689
17,4324
17,6862
17,8085
17,8283
17,1417
17,3383
17,3221
17,7010
17,3182
17,5882
17,1164
17,3559
17,6242
17,6601
17,8953
17,7993
17,9856
17,5624
17,4747
17,7662
17,1674
17,3700
18,2298
18,3182
18,6601
19,3700
19,4747
20,3559
21,1674
21,5951
21,5624
21,6601
21,9856
21,6242
21,5276
21,2873
21,3182
21,7993
21,8312
21,1133
21,7662
21,3383
22,6862
22,1133
22,3559
22,6601
22,9771
22,7993
22,7310
22,5030
22,6357
22,1164
22,1689
22,7010
22,8085
22,3221
22,6440
22,2298
22,4726
22,3596
22,2791
22,1076
22,3121
23,4789
23,8312
23,7010
23,2182
23,3559
23,2791
23,8953
23,3700
23,8810
23,4324
23,9771
23,7662
23,9856
23,7272
23,3121
23,7113
23,2873
23,1674
23,4747
23,1076
23,8283
24,3596
24,8810
24,5624
25,4191
25,1164
25,5276
25,3182
26,4324
26,9856
26,5951
26,6440
26,9771
26,7272
26,4191
26,7310
26,4726
26,5624
26,1689
26,2298
26,7993
26,3221
26,8283
26,1076
26,7662
26,7113
26,4789
26,6601
26,1361
26,3383
26,3700
27,7804
27,1164
28,1133
28,1076
28,4789
28,5030
28,1164
28,5882
28,6862
28,6601
28,3559
28,3121
28,1361
28,4191
28,8085
29,8312
29,3221
29,1674
29,7272
29,8810
29,5030
29,9856
29,1417
29,6862
29,3596
29,1164
29,8283
29,7010
29,8953
29,3639
29,5624
29,7662
29,8085
29,7310
30,3559
30,3182
30,4324
30,1361
