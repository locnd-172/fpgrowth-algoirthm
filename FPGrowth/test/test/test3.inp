1076 1133 1164 1689 2298 2791 3121 3221 3559 3596 4726 5030 6357 6440 6601 6862 7010 7310 7993 8085 9771 
1076 1674 2182 2791 2873 3121 3559 3700 4324 4747 4789 7010 7113 7272 7662 8283 8312 8810 8953 9771 9856 
3596 5624 8810 
1164 3182 4191 5276 
1076 1361 1689 2298 3221 3383 3700 4191 4324 4726 4789 5624 5951 6440 6601 7113 7272 7310 7662 7993 8283 9771 9856 
1164 7804 
1076 1133 1164 1361 3121 3559 4191 4789 5030 5882 6601 6862 8085 
1164 1417 1674 3221 3596 3639 5030 5624 6862 7010 7272 7310 7662 8085 8283 8312 8810 8953 9856 
1361 3182 3559 4324 
1076 1133 1361 2182 3121 3221 3639 3700 4726 4789 5276 6601 6862 7272 7804 8312 8651 
2791 3383 3596 5030 6357 7662 8283 9771 
1164 1361 4726 5030 6242 8312 8953 9856 
1361 1674 1689 2182 2873 3221 3559 3639 3700 4726 4747 4789 5030 5276 5882 6357 6440 6601 7272 7662 7993 8283 9856 
1076 7010 
2873 
1674 1689 2298 2873 3121 3221 3383 3639 4324 5276 5882 6242 6440 6601 6862 7310 7804 8651 8810 9856 
1164 1417 1674 3182 3221 3383 3559 3700 4324 4747 5624 5882 6242 6601 6862 7010 7662 7993 8085 8283 8953 9856 
2298 3182 6601 
3700 4747 
4324 5030 5951 
1076 2791 3182 3221 3559 4191 4789 5276 6242 6440 6601 7113 7804 8810 9771 
1076 1417 1689 2182 2298 3121 3383 3700 4324 4726 4789 5276 5624 6440 7272 8283 8312 9771 
1076 1133 1361 1674 1689 3182 3221 3383 4191 4726 4789 6242 7010 7310 8810 9856 
1164 1417 1689 2873 3383 3639 3700 4747 4789 5882 5951 6357 6601 6862 7310 7662 7804 7993 8085 8283 8312 
1133 1164 1689 2791 2873 3121 3221 3383 3559 3596 4324 4747 7010 7272 
1076 1689 3559 3596 3639 3700 4324 5276 5882 6242 6357 6440 6862 7113 7310 7662 7740 7993 8283 9771 9856 
1164 1417 2298 2791 3182 3559 3639 4324 4789 5030 5276 5951 6601 7113 7272 7310 7662 8312 
1133 1361 1417 3121 3559 4747 5030 5276 5624 6242 6440 6862 7662 8810 9771 
3559 
1133 1674 2873 3182 3383 5276 5624 5951 6242 6601 7662 7993 8312 9856 
