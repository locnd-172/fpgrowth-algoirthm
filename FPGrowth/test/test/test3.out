1133:7
1361:8
1417:6
1674:7
1689:9
2298:6
2791:6
2873:7
3121:8
3182:8
3383:9
3596:6
3639:7
3700:9
4726:7
4747:7
5030:9
5624:7
5882:6
6242:8
6440:8
6862:9
7010:7
7272:8
7310:8
7993:7
8283:9
8312:8
8810:7
9771:8
1076:10
1164:10
3221:10
3559:12
4324:10
4789:10
5276:10
6601:12
7662:11
9856:10
1076 4789:7
1076 9771:6
1674 9856:7
1689 3221:6
1689 3383:6
1689 6440:6
1689 7310:6
3221 6601:7
3221 9856:6
3559 4324:6
3559 6601:6
3559 7662:6
3700 4789:6
3700 7662:6
3700 8283:7
4789 6601:7
4789 7272:6
5276 6440:6
5276 6601:6
6242 9856:6
6440 9771:6
6601 6862:6
6601 7662:6
6601 7993:6
7662 7993:6
7662 8283:8
7662 9856:7
8283 9856:6
3700 7662 8283:6
7662 8283 9856:6
